src/main/java/uk/gov/hmcts/reform/wacaseeventhandler/clients/DatabaseMessageConsumer.java

```diff
@@ -159,24 +159,24 @@ private Optional<MessageUpdateRetry> processMessage(CaseEventMessage caseEventMe
     }
 
     private Optional<MessageUpdateRetry> processException(FeignException fce, CaseEventMessage caseEventMessage) {
-        boolean retryableError = false;
+        boolean isNonRetryableError = true;
         try {
             final HttpStatus httpStatus = HttpStatus.valueOf(fce.status());
-            retryableError = RestExceptionCategory.isRetryableError(httpStatus);
+            isNonRetryableError = UnprocessableHttpErrors.isNonRetryableError(httpStatus);
         } catch (IllegalArgumentException iae) {
             if (fce instanceof RetryableException) {
```

Comment:
I know you didn't add this but can an IllegalArgumentException be a RetryableException?  Seems not.

Reply:
yes this is existing piece of code, was also wondering why we have added it so left as is for now